src/auth/jwt.strategy.ts

```diff
@@ -1,21 +1,27 @@
 import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
 import { PassportStrategy } from '@nestjs/passport';
 import { ExtractJwt, Strategy } from 'passport-jwt';
+import { EmployeesService } from 'src/employees/employees.service';
 
 @Injectable()
 export class JwtStrategy extends PassportStrategy(Strategy) {
-  constructor() {
+  constructor(
+    private readonly employeesService: EmployeesService,
+    private configService: ConfigService,
+  ) {
     super({
       jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
       ignoreExpiration: false,
-      secretOrKey: process.env.SECRET, //  TODO: get from env var
+      secretOrKey: `SeCR3TK3YF0RAp1@vhris`, //  TODO: get from env var
```

Comment:
@vision-jm 
kung walang downside na naka-expose sa repo ung secret key, baka mas ok na nasa env file nalang sya, what do you think?